packages/kbn-management/cards_navigation/src/types.ts

```diff
@@ -34,9 +67,15 @@ export interface ManagementAppProps {
 }
 
 export interface AppDefinition {
-  category: string;
+  category: typeof appCategories[keyof typeof appCategories];
   description: string;
-  icon: React.ReactElement;
+  icon: EuiIconProps['type'];
+}
+
+export interface CardNavExtensionDefinition<N extends boolean> extends AppDefinition {
+  noVerify: N;
```

Comment:
I think the introduction of this noVerify prop makes sense, but I was thinking whether it would make sense to rename it something more like "skipValidation" or something like that.. wdyt?

Also was wonderig if we should add some comments on this to have better documentation with intelisense.

Reply:
I like the idea of adding in comments to match with the existing intelisense, I'll give it a try along with adopting `skipValidation` for more clarity.